### DISCUSSION TITLE
Game crash at start

### DISCUSSION THREAD

**Discussion by FFFemboy** [ErinaSugino/Starbound-SBR-Mod#147] [Trouble Shooting] [answered]
I followed the guide line by line but the game crashes, checked the log file and here is the error line that crash the game:
`[00:40:35.653] [Error] Application: exception thrown, shutting down: (StatusEffectDatabaseException) Duplicate stat effect named 'sexbound_arousal_debuff1', config file '/stats/effects/sexbound_arousal_debuff1/sexbound_arousal_debuff1.statuseffect'`
problem: im 100% sure i only installed sxb reborn and no other mods has a stat folder that might create a conflict with sxb (aside from the .pak ones because i cant check inside)
I tried disabling some 1 at a time, checking for updates but i'm still here with crash at start.
here is the mods list (this is my starbound\mods folder):
getridofthatunderwear 3.5.0
MonsterHormones
MorePoses_0.1.1
SBR Aphrodites Bow patch
sexbound
sexbound-defeat-2.7.0.src
The Pygophilia Upgrade POV Complete
Lustbound_Aphrodites_Bow_Addon.pak
Lustbound_Omnibrowser_Addon.pak
mods_go_here
sexbound-aphrodites-bow-6.2.0.pak
sexbound-EBsLovelyWhips1.1.0.pak
sexbound_brothel 1.2.pak
Sexbound_KawaiiStuff+ Supports.pak
starbound-maid4u-1.0.0.pak
starbound-omnibrowser-1.6.0.pak
SxB - API POV Redux 4.2.0.pak
SxB - Teratophilia 0.4.8.pak
SxB_-_TemTems_Teratoscenes.pak
TTPPv2.7.pak

**Comment by ErinaSugino** [comment]
Are you sure that you have downloaded the newest version of the mod?

**Reply by FFFemboy** [reply]
updated every single one of them.

**Reply by ErinaSugino** [reply]
Do you have the mod as source version? If so, have you updated it recently by just merging the new folder with the old one?

**Reply by FFFemboy** [reply]
yes i did, what have i done wrong?

**Reply by ErinaSugino** [reply, marked as answer]
The issue comes from the fact that I have moved files regarding the status effects for arousal and heat. They are now grouped in a new sub-folder. If you just override the old folder with the new one, however, this only *adds or changes* files, not remove them. So now you have the old AND the new files for the same status effects, which is causing the error. You can fix this by manually deleting the old SBR folder instead of overwriting it.

**Reply by FFFemboy** [reply]
thanks, everything is fixed :D
